package.json

```diff
@@ -81,6 +81,7 @@
     "html-webpack-plugin-v4": "npm:html-webpack-plugin@^4.5.0",
     "html-webpack-plugin-v5": "npm:html-webpack-plugin@^5.0.0-alpha.14",
     "husky": "^4.2.5",
+    "idb": "^6.0.0",
```

Comment:
Just double-checking—what necessitates adding in the `idb` dependency at the top-level?Is it there for the test server, for creating our `build/` bundles, or some other purpose?

Reply:
For the test server. Correct me if I'm wrong but to use it in the tests, it needs to be included at the top-level. That is why I added the 'node_modules' as a directory in sw-bundle.js. Not sure if there is another way to do it.

Reply:
👍